src/app/shared/util/metrics.util.ts

```diff
@@ -1,23 +1,8 @@
-import { PodcastModel, EpisodeModel, RouterModel,
+import { EpisodeModel, RouterModel,
   PodcastMetricsModel, EpisodeMetricsModel, getMetricsProperty,
   IntervalModel, INTERVAL_HOURLY, INTERVAL_DAILY, INTERVAL_WEEKLY, INTERVAL_MONTHLY } from '../../ngrx';
 import * as dateUtil from './date/date.util';
 
-export const filterPodcasts = (filter: RouterModel, podcasts: PodcastModel[]): PodcastModel => {
```

Comment:
This filtering happens in the selectors now.

Reply:
👍